CHANGELOG.md

```diff
@@ -3,14 +3,6 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## [1.0dev] - 2020-04-XX
```

Comment:
Aren't we in may already?

Reply:
Fixed in https://github.com/nf-core/viralrecon/pull/106/commits/ebaf52c689e6099bb2479fa38e75b4ccbd4c2edc and https://github.com/nf-core/viralrecon/pull/106/commits/5c8cc7c7b089a849db096b1d305d5fe5b4c6a8c5